test/token/ERC721/ERC721MintBurn.behavior.js

```diff
@@ -91,8 +91,10 @@ function shouldBehaveLikeMintAndBurnERC721Token (accounts) {
           logs = result.logs;
         });
 
-        it('clears the approval', async function () {
-          (await this.token.getApproved(tokenId)).should.be.equal(ZERO_ADDRESS);
+        context('when the token ID does not exist', function () {
+          it('reverts', async function () {
```

Comment:
I find the current phrasing a bit confusing: maybe we could merge the `context` and `it` blocks, and change the description to something like 'causes getApproved to revert'?

We also may want to add a test block for `getApproved`, explicitly checking for this, though that could be considered part of #1148.